src/property-filter/styles.scss

```
@@ -43,21 +43,40 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  &-line {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
+  &-form {
+    display: grid;
+    grid-template-columns: max-content minmax(awsui.$space-xl, auto) minmax(200px, min-content);
```

I believe, it should be: 

```suggestion
    grid-template-columns: max-content minmax(awsui.$space-xl, 1fr) minmax(200px, min-content);
```